sdk/dotnet/Pulumi/Resources/InputArgs.cs

```diff
@@ -17,8 +17,6 @@ public abstract class InputArgs
     {
         private readonly ImmutableArray<InputInfo> _inputInfos;
 
-        private protected abstract void ValidateMember(Type memberType, string fullName);
```

Comment:
In the past this has been used to validate members as follows:

- `ResourceArgs` members must be `Input<T>`, but this was relaxed in #6522 to support plain values
- `InvokeArgs` members must not be `Input<T>`, but relaxing this for `Call` which does allow `Input<T>`

I've removed `ValidateMember` since there's no longer any validations to do.

Alternatively, we could consider adding new `CallArgs` with empty `ValidateMember` (and potentially `CallOptions`), but these would basically the same as `InvokeArgs` and `InvokeOptions`.